tests/draft2019-09/if-then-else.json

```diff
@@ -184,5 +184,37 @@
                 "valid": true
             }
         ]
+    },
+    {
+        "description": "if with boolean schema true",
+        "schema": { "if": true, "then": { "const": "then" }, "else": { "const": "else" } },
```

Comment:
Could you reformat this, typically in the suite stuff is broken up on multiple lines even if it all fits on one, just so its easy to see each individual validator present

Reply:
done!